### PR TITLE
[WASH-930] TozID login for iOS

### PR DESCRIPTION
**Pull request by carrala**
- Adds login method for iOS, which uses the updated e3db-swift and returns a Client
- Also updated the permissions in info.plist so that http requests succeed

### CODE REVIEW

**Review by sjonz — APPROVED** [review]

**Comment by sjonz** [comment]
Just a suggestion - make a follow up task (e.g. Jira Subtask) to make sure we unpin the e3db dependency.

**Review by efabens — APPROVED** [review]

**Review by rosasck — APPROVED** [review]

File: lib/tozny_model.dart
Comment (Mrhea) [inline lib/tozny_model.dart:222]: After the changes to Dart was it confirmed that the Android plugin still runs? Nothing here looks like a breaking change, but just want to make sure

**Review by Mrhea — APPROVED** [review]

File: ios/Classes/E3dbSerializer.swift
Comment (galxy25) [inline ios/Classes/E3dbSerializer.swift:8]: what was broken is now fixed 🧙🏽‍♀️ 🧙🏽‍♂️

**Review by galxy25 — APPROVED** [review]

File: lib/tozny_model.dart
Comment (carrala) [reply lib/tozny_model.dart:222]: good point. i'll verify before merging.

File: example/ios/Runner/Info.plist
Comment (pirtleshell) [inline example/ios/Runner/Info.plist]: just flagging that according to the docs this configuration flag will need to be justified to Apple to get this code in the app store. i'm not familiar enough with this project to know if that is a concern or if this is just some debug code to allow non-https protocol requests. just noting it may come up

>You must supply a justification during App Store review if you set the key’s value to YES, as described in Provide Justification for Exceptions. Use this key with caution because it significantly reduces the security of your app. In most cases, it’s better to upgrade your servers to meet the requirements imposed by ATS, or at least to use a narrower exception.

https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloads

**Review by pirtleshell — APPROVED** [review]
approved assuming there are no concerns around `NSAllowsArbitraryLoads`

File: lib/tozny_model.dart
Comment (Mrhea) [reply lib/tozny_model.dart:222]: Android plugin confirmed to work

File: example/ios/Runner/Info.plist
Comment (galxy25) [reply example/ios/Runner/Info.plist]: updated to add this context for future readers

**Pull request merged**
